src/lib/server/database/models/index.ts

```diff
@@ -51,13 +51,34 @@ export const confession = app.table(
         approvedAt: timestamp('approved_at', { withTimezone: true }).defaultNow(),
         authorId: bigint('author_id', { mode: 'bigint' }).notNull(),
         content: text('content').notNull(),
+        attachmentId: bigint('attachment_id', { mode: 'bigint' }).references(() => attachmentData.attachmentId),
     },
-    ({ confessionId, channelId }) => [uniqueIndex('confession_to_channel_unique_idx').on(confessionId, channelId)],
+    ({ confessionId, channelId, attachmentId }) => [
+        uniqueIndex('confession_to_channel_unique_idx').on(confessionId, channelId),
+        uniqueIndex('confession_to_attachment_unique_idx').on(confessionId, attachmentId),
+    ],
 );
 
 export type Confession = typeof confession.$inferSelect;
 export type NewConfession = typeof confession.$inferInsert;
 
 export const confessionRelations = relations(confession, ({ one }) => ({
     channel: one(channel, { fields: [confession.channelId], references: [channel.id] }),
+    attachment: one(attachmentData, { fields: [confession.attachmentId], references: [attachmentData.attachmentId] }),
 }));
+
+export const attachmentData = app.table('attachment_data', {
```

Comment:
I prefer a more concise table name.

```suggestion
export const attachment = app.table('attachment', {
```

Reply:
My concern with naming the table was the possibility of collision with the `attachment` object that I frequently declare in the code for the attachment. Now I think about it, though, I think I could do just as well with a renamed import. Lemme see.

Reply:
Never mind on the renamed import, that would look messy and confusing. I'll just adjust the naming of the `attachment` object.